sdk/storage/azblob/README.md

```diff
@@ -1,443 +1,276 @@
+client, err := azblob.NewClientFromConnectionString(connectionString, nil)
+// TODO: handle error
 
-var responseErr *azcore.ResponseError
-errors.As(err, &responseErr)
+// try to delete the container, avoiding any potential race conditions with an in-progress or completed deletion
+_, err = client.DeleteContainer(context.TODO(), containerName, nil)
 
-if responseErr != nil {
-fmt.Println("ErrorCode: " + responseErr.ErrorCode + ", StatusCode: " + strconv.Itoa(responseErr.StatusCode))
-} else {
-fmt.Println(err.Error())
+if respErr := &(azcore.ResponseError{}); errors.As(err, &respErr) {
+	if respErr.ErrorCode == string(bloberror.ContainerBeingDeleted) || respErr.ErrorCode == string(bloberror.ContainerNotFound) {
+		// ignore any errors if the container is being deleted or already has been deleted
+	}
+} else if err != nil {
+	// TODO: some other error
 }
 ```
 
-### Logging
-
-This module uses the classification based logging implementation in azcore. To turn on logging
-set `AZURE_SDK_GO_LOGGING` to `all`.
+## Next steps
 
-If you only want to include logs for `azblob`, you must create your own logger and set the log classification
-as `LogCredential`.
+Get started with our [Blob samples][samples].  They contain complete examples of the above snippets and more.
 
-To obtain more detailed logging, including request/response bodies and header values, make sure to leave the logger as
-default or enable the `LogRequest` and/or `LogResponse` classificatons. A logger that only includes credential logs can
-be like the following:
+### Specialized clients
 
-```golang
-import azlog "github.com/Azure/azure-sdk-for-go/sdk/azcore/log"
-// Set log to output to the console
-azlog.SetListener(func (cls azlog.Classification, msg string) {
-fmt.Println(msg) // printing log out to the console
-})
+The Azure Blob Storage SDK for Go also provides specialized clients in various subpackages.
+Use these clients when you need to interact with a specific kind of blob.
+Learn more about the various types of blobs from the following links.
 
-// Includes only requests and responses in credential logs
-azlog.SetClassifications(azlog.Request, azlog.Response)
-```
+- [appendblob][append_blob] - [REST docs](https://docs.microsoft.com/rest/api/storageservices/understanding-block-blobs--append-blobs--and-page-blobs#about-append-blobs)
+- [blockblob][block_blob] - [REST docs](https://docs.microsoft.com/rest/api/storageservices/understanding-block-blobs--append-blobs--and-page-blobs#about-block-blobs)
+- [pageblob][page_blob] - [REST docs](https://docs.microsoft.com/rest/api/storageservices/understanding-block-blobs--append-blobs--and-page-blobs#about-page-blobs)
 
-> CAUTION: logs from credentials contain sensitive information.
-> These logs must be protected to avoid compromising account security.
->
+The [blob][blob] client contains APIs common to all blob types.  This includes APIs for deleting and undeleting a blob, setting metadata, and more.
```

Comment:
```suggestion
The [blob][blob] package contains APIs common to all blob types.  This includes APIs for deleting and undeleting a blob, setting metadata, and more.
```
(the link just takes me the package folder so I'm assuming you meant package, like the 'lease' package below?)

Reply:
Or should these point to the pkg.go.dev docs instead?

Reply:
Agreed that this should point to `pkg.go.dev`.

For the wording, I was trying to distinguish between the fact that `blob` exposes a client and that `lease` exposes multiple things.  This might not age well though, so I'll update them all to be consistent.

Reply:
Ugh ok I remember that I pointed to the directories because this module hasn't been published yet.  Will have to update this later.

Reply:
I did for all except `lease` and `sas` as they don't have a single `client.go` file.